src/btree/bt_sync.c

```diff
@@ -109,44 +109,69 @@ __sync_ref_is_obsolete(WT_SESSION_IMPL *session, WT_REF *ref)
 {
     WT_ADDR *addr;
     WT_CELL_UNPACK vpack;
+    WT_PAGE_MODIFY *mod;
+    bool obsolete;
 
     /* Ignore root pages as they can never be deleted. */
-    if (__wt_ref_is_root(ref))
-        return (false);
-
-    /* Guard against marking an in-memory page to be deleted. */
-    if (ref->state != WT_REF_DISK)
```

Comment:
These lines appear to have been lost in the change.
I'm assuming this PR and ticket are only to add verbose output and no other functional changes.

Reply:
I agree with @ddanderson here. There are a lot of non-verbose related changes that are concerning to me. Please address that and then I'll do another review.